script/deploy/zksync/DeployLiFiDEXAggregator.s.sol

```diff
@@ -23,13 +23,15 @@ contract DeployScript is DeployScriptBase {
 
     function getConstructorArgs() internal override returns (bytes memory) {
         string memory path = string.concat(root, "/config/global.json");
-        string memory json = vm.readFile(path);
 
         address[] memory priviledgedUsers = new address[](1);
-        priviledgedUsers[0] = json.readAddress(".pauserWallet");
+        priviledgedUsers[0] = _getConfigContractAddress(path, ".pauserWallet");
```

Comment:
this is wrong. the helper function must only be used for contract addresses, not for EOAs. This will not work.

Reply:
FYI @mirooon

Reply:
yep

Reply:
@0xDEnYO fixed